dev-util/makeheaders/makeheaders-20220411.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Tool that automatically generates .h files for a C or C++ programming project"
+HOMEPAGE="https://www.hwaci.com/sw/mkhdr/"
+
+SRC_URI="
+	https://fossil-scm.org/home/raw/ca90c1e2460d79e48f97c2aac60a39bcb8024e394de7cff5164c9dcbc96ce529?at=makeheaders.c -> makeheaders-${PV}.c
+	https://fossil-scm.org/home/raw/262696252dc50250c896c90cc240dcd946614b9c7727902aa7606640507e9231?at=makeheaders.html  -> makeheaders-${PV}.html
```

Comment:
If it works always like that I'd suggest to use something like
`COMMIT-C="ca90c1e2460d79e48f97c2aac60a39bcb8024e394de7cff5164c9dcbc96ce529"`
and
`COMMIT-HTML="262696252dc50250c896c90cc240dcd946614b9c7727902aa7606640507e9231"`

just under the inherit and use

```
SRC_URI="
	https://fossil-scm.org/home/raw/ca90c1e2460d79e48f97${COMMIT-C}?at=makeheaders.c -> makeheaders-${PV}.c
	https://fossil-scm.org/home/raw/${COMMIT-HTML}?at=makeheaders.html  -> makeheaders-${PV}.html
```
In this way you can just change COMMIT's values without always change the entire link

Reply:
Done in https://www.github.com/gentoo/gentoo/pull/28424